drivers/AVR/canard_AVR.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016 UAVCAN Team
+ *
+ * Distributed under the MIT License, available in the file LICENSE.
+ *
+ */
+
+#ifndef CANARD_AVR_H
+#define CANARD_AVR_H
+
+#include <canard.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * \ingroup   communication
+ * \defgroup  canard_avr_interface Libcanard CAN Interface for AVR microcontrollers
+ * \brief     Interface for Libcanard CAN interaction with AVR microcontrollers
+ *
+ * \author    Matthias Renner <rennerm@ethz.ch>
+ * \author    ETH Zuerich Robotics Systems Lab (http://http://www.rsl.ethz.ch/)
+ *
+ * \version   0.1
+ */
+
+
+/**
+ * @ingroup canard_avr_interface
+ * @brief Initialize CAN interface on AVR microcontroller.
+ * @warning Enables interrupts!
+ *
+ * @param [in] id  CAN ID for hardware filter (to be implemented)
+ *
+ * @retval     0   Successfully initialized.
+ */
+int canardAVRInit(uint32_t id);
```

Comment:
You can't pass node ID in the driver initialization function, because in case of dynamic allocation, the node ID is not known at the time of driver initialization. I propose to add a dedicated function for configuration of the node ID, e.g. `canardAVRConfigureAcceptanceFilters(uint8_t node_id)` (notice that we use `uint8_t` for node ID, not `uint32_t`). Seeing as this functionality is, anyway, not implemented, this function may be added in later revisions.

CAN bitrate should be configurable. I suggest to add an argument for that to the initialization function, preferably in bits/sec.